Features/Stories/Fieldtypes/CheckboxFieldType.feature

```diff
@@ -0,0 +1,30 @@
+Feature: Test the validations done on fields from Editorial Interface - Checkbox fieldtype
+
+    @javascript
+    Scenario: Publish with checkbox checked
+        Given I am logged in as admin on PlatformUI
+        And a Content Type exists with identifier "Checkbox" in Group with identifier "Content" with fields:
```

Comment:
It's not the point. The fact that it is part of this or that content type group has no incidence on this scenario.